config/config.go

```diff
@@ -86,6 +86,10 @@ func (c *Config) fillDefaultValues() {
 		"pitaya.modules.bindingstorage.etcd.prefix":             "pitaya/",
 		"pitaya.modules.bindingstorage.etcd.dialtimeout":        "5s",
 		"pitaya.modules.bindingstorage.etcd.leasettl":           "1h",
+		"pitaya.groups.etcd.endpoints":                          "localhost:2379",
+		"pitaya.groups.etcd.prefix":                             "pitaya/",
+		"pitaya.groups.etcd.dialtimeout":                        "5s",
+		"pitaya.groups.etcd.leasettl":                           "1h",
```

Comment:
Don't forget to add these configs to the docs

Reply:
Done!